### PR TITLE
PXB-2920 Fix misspelling in versioning scheme (8.0)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/xtrabackup-versioning-scheme.md
renamed:    docs/xtrabackup-versioning-scheme.md -> docs/xtrabackup-version-numbers.md
modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/xtrabackup-version-numbers.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Understand version numbers
+
+A version number identifies the product release and any updates. The product contains the latest Generally Available (GA) features at the time of that release. If a feature must be modified after the release, the modification can be either added as a patch fix or included in a new release.
```
Comment (patrickbirch) [inline docs/xtrabackup-version-numbers.md]: added as a patch fix or ...
PXB doesn't do patch fixes.

File: docs/xtrabackup-version-numbers.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Percona uses semantic version numbering, which follows the pattern of base version and minor version. Percona assigns unique, non-negative integers in increasing order for each version release.
+
+The version numbers for Percona XtraBackup 8.0.30-23 define the following information:
+
+* Base version - the leftmost number indicates the version of the Percona XtraBackup that is based on [MySQL 8.0](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/). A base version increase resets the minor version to 0. 
+
+* Minor version - an internal number that denotes the version of the software. A minor version increases by one each time the Percona XtraBackup is released.
+
+Percona XtraBackup 8.0.28-20 and 8.0.28-21 are multiple releases based on MySQL 8.0.28.
```
Comment (patrickbirch) [inline docs/xtrabackup-version-numbers.md:32]: Percona Server for MySQL

File: docs/xtrabackup-version-numbers.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup-version-numbers.md]: updated

File: docs/xtrabackup-version-numbers.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+
```
Comment (patrickbirch) [inline docs/xtrabackup-version-numbers.md:36]: Add a reference to the blog
https://www.percona.com/blog/2020/08/18/aligning-percona-xtrabackup-versions-with-percona-server-for-mysql/
and mention that the version numbering changed after 8.0.14

Comment (alina-derkach-oaza) [reply docs/xtrabackup-version-numbers.md:36]: Updated

File: docs/xtrabackup-version-numbers.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+The version numbers for Percona XtraBackup 8.0.30-23 define the following information:
+
+* Base version - the leftmost number indicates the version of the Percona XtraBackup that is based on [MySQL 8.0](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/). A base version increase resets the minor version to 0. 
+
+* Minor version - an internal number that denotes the version of the software. A minor version increases by one each time the Percona XtraBackup is released.
+
+Percona XtraBackup 8.0.28-20 and 8.0.28-21 are multiple releases based on MySQL 8.0.28.
+
+Note that Percona XtraBackup numbering changed after the 8.0.14 version to align Percona XtraBackup versions with Percona Server and MySQL. Find more information in [Aligning Percona XtraBackup Versions with Percona Server for MySQL](https://www.percona.com/blog/2020/08/18/aligning-percona-xtrabackup-versions-with-percona-server-for-mysql/) blog post.
```
Comment (patrickbirch) [inline docs/xtrabackup-version-numbers.md]: Move this paragraph to line 26.

Comment (alina-derkach-oaza) [reply docs/xtrabackup-version-numbers.md]: updated

**Review by patrickbirch — APPROVED** [review]

**Pull request merged**